scipy/stats/_multivariate.py

```diff
@@ -2063,7 +2056,12 @@ def _standard_rvs(self, n, shape, dim, df, random_state):
             Dimension of the scale matrix
         df : int
             Degrees of freedom
-        random_state : {`~np.random.RandomState`, `~np.random.Generator`}
+        random_state : {None, int, `numpy.random.Generator`}, optional
+            If `seed` is None the `numpy.random.Generator` singleton is used.
+            If `seed` is an int, a new ``Generator`` instance is used,
+            seeded with `seed`.
+            If `seed` is already a ``Generator`` instance then that instance is
+            used.
```

Comment:
The old version is more correct. By the time this method gets called, the value has already been coerced into either a `RandomState` or a `Generator`.